qualang_tools/config/examples/resonator.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+
+from config.configuration import *
+from config.components import *
+from config.builder import ConfigBuilder
+
+cont = Controller("con1")
+
+res = ReadoutResonator("res1", outputs=[cont.analog_output(0), cont.analog_output(1)],
+                       inputs=[cont.analog_input(0),cont.analog_input(1)],
+                       intermediate_frequency=2e6)
+res.lo_frequency = 4e9
```

Comment:
I wonder where those examples are supposed to be. 

Also, probably worth adding the examples as tests, to validate that the generated qua_config is correct.

Reply:
I think that the examples should be in the `qua-libs` git. 
I just finished moving the bakery examples there.
Please move it there (or convince me otherwise)

Reply:
I added some basic tests (not exhaustive though) for transmon and resonator configs. The examples are moved to `config-builder-examples` branch in qua-libs